src/Movies/Movies.js

```diff
@@ -8,10 +8,35 @@ class Movies extends Component {
     super()
     this.state = {
       movies: [],
+      searchInput: "",
       error: null
     }
   }
 
+  handleChange = (event) => {
+    this.setState({ searchInput: event.target.value })
+  }
+
+  formatSearchInput = () => {
+    let lowerCaseInput = this.state.searchInput.toLowerCase()
+    let formattedInput = lowerCaseInput.charAt(0).toUpperCase() + lowerCaseInput.slice(1)
+    return formattedInput
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault()
+    let inputTitle = this.formatSearchInput()
+    let filteredMovies = this.state.movies.filter(movie => {
+      return movie.title.includes(inputTitle)
+    })
+    if (!filteredMovies.length) {
+      alert("No results match this search")
```

Comment:
Worked on a couple different methods to avoid using an alert but none were as good functionally as this.